src/app/search/search.component.ts

```diff
@@ -389,16 +390,19 @@ export class SearchComponent implements OnInit, OnDestroy {
   }
 
   updateSideBar(value: string) {
-    ELASTIC_SEARCH_CLIENT.search({
-      index: 'tools',
-      type: 'entry',
-      body: value,
-    })
-      .then((hits) => {
-        this.setupAllBuckets(hits);
-        this.setupOrderBuckets();
-      })
-      .catch((error) => console.log(error));
+    this.extendedGA4GHService.toolsIndexSearch(value).subscribe(
+      (hits: any) => {
+        if (hits) {
+          this.setupAllBuckets(hits);
```

Comment:
Without understanding entirely, if there aren't any hits, don't you need to "clear"? Although look at `setupAllBuckets()`, it would have failed before as well.

Reply:
ya, that's true. i put this hit check for testing purposes only, it's never falsey. it won't affect anything in practice but i suppose i should find a better solution